src/lasp_core.erl

```diff
@@ -554,10 +560,8 @@ bind_to(AccId, Id, Store, BindFun) ->
     bind_to(AccId, Id, Store, BindFun, ?READ).
 
 bind_to(AccId, Id, Store, BindFun, ReadFun) ->
-    Fun = fun({_, _, _, V}) ->
-        {ok, _} = BindFun(AccId, V, Store)
-    end,
-    lasp_process:start_link([[{Id, ReadFun}], Fun]).
+    TransFun = fun({_, _, _, V}) -> V end,
```

Comment:
Yes, `bind_to` originally had different behavior, but now it's similar to a map.  It isn't *precisely* map, because it works for all data types, where map only operates on collection types.